### ISSUE TITLE
url_launcher says status 'awaiting'

### ISSUE THREAD

**Issue by mit-mit** [dart-lang/pub-dartlang-dart#917]
https://pub.dartlang.org/packages/url_launcher#-analysis-tab-

**Comment by isoos** [comment]
There was an issue during fetching the dependencies:

```
2018-01-12 16:10:27.064 CET
pub.analyzer.pana: Failed (v0.9.0+1) - url_launcher/2.0.1 Error: ProcessException: No such file or directory Command: /flutter/bin/flutter packages pub upgrade --verbosity io --no-precompile Stack: #0 runProc (package:pana/src/utils.dart:36:17) <asynchronous suspension> #1 FlutterSdk._execUpgrade (package:pana/src/sdk_env.dart:128:7) 
```

**Comment by isoos** [comment]
I'm not yet sure whether it was a transient issue or something repeated, because subsequent (daily) retries failed. The reason for that is: the throttling logic seems to be too strict, we'll need to change that.

```
2018-01-13 16:30:43.026 CET pub.analyzer.pana:
Task url_launcher 2.0.1 skipped because: Previous analysis completed after task's trigger timestamp.
```

Trying to repeat it with local flutter 0.0.20...

**Comment by jakobr-google** [comment]
Analysis server logs are full of errors trying to clone the flutter repo. I've sent out https://github.com/dart-lang/pub-dartlang-dart/pull/918 to fix that.

**Comment by jakobr-google** [comment]
(...and by 'full', I mean once per instance restart...)

**Comment by isoos** [comment]
Yeah, and it looks like it started on `2018-01-12 13:25:03.522`.
An analysis of the same setup completed the day before (on a previous version of url_launcher): https://pub.dartlang.org/packages/url_launcher/versions/2.0.0#-analysis-tab-

**Comment by isoos** [comment]
I guess the Flutter repo changed their tags?

**Comment by jakobr-google** [comment]
Looks like it, yeah. My slightly old checkout has 0.0.20, etc. tags, but new clones don't.

**Comment by isoos** [comment]
In addition to #918, I'd also send a fix for the retry logic, otherwise a new release won't clear it right away.

**Comment by mit-mit** [comment]
@isoos what is the status on this? I still see awaiting...

**Comment by jakobr-google** [comment]
We're (a)waiting on me deploying a new release.

**Comment by jakobr-google** [comment]
url_launcher is fixed now, and the remaining should be corrected once the new instances get to them.

**Comment by kevmoo** [comment]
Still seeing this here - https://pub.dartlang.org/packages/dart_language_server#-analysis-tab- - I think

Still `awaiting` a deploy?

**Comment by jakobr-google** [comment]
The new version was deployed. Looks like this package is failing because dartanalyzer doesn't like it.

**Comment by kevmoo** [comment]
Ah! it shouldn't be stuck at 'awaiting' then, right?

**Comment by jakobr-google** [comment]
Nah, it'd be better if it failed, I think. Right now, I think we keep retrying failed packages in the hope that it's a temporary tool failure. We should trust the tools more. Could you open a new issue for this one, please?
